test-requirements.txt

```diff
@@ -1,4 +1,4 @@
-pytest>=3.6
+pytest>=3.6,<5.4
```

Comment:
What errors did you get with pytest 5.4? I just tried locally with pytest 5.4.1 and it worked for me.

Asking because I'd prefer not to pin it if we don't have to, and if we do I'd like to know why so I have some sense of how difficult it will be to update.

Reply:
```
INTERNALERROR>   File "/Users/kevinmcelwee/anaconda3/envs/py36/lib/python3.6/site-packages/pytest_sugar.py", line 176, in pytest_configure
INTERNALERROR>     sugar_reporter = SugarTerminalReporter(standard_reporter)
INTERNALERROR>   File "/Users/kevinmcelwee/anaconda3/envs/py36/lib/python3.6/site-packages/pytest_sugar.py", line 214, in __init__
INTERNALERROR>     self.writer = self._tw
INTERNALERROR> AttributeError: can't set attribute
```

It looks like a Java error that comes from `pytest_sugar`. Going to `pytest < 5.4` fixed it for me. Thoughts?

Reply:
Please try uninstaling `pytest-sugar` and checking with the latest pytest. If that works, then remove it from the dev requirements and unpin pytest. We don't need it, it just makes the test output nicer (and I think it started causing problems on travis-ci recently too).

Reply:
perfect, it's all good